debian/clearwater-socket-factory.postinst

```diff
@@ -0,0 +1,162 @@
+#!/bin/sh
```

Comment:
Many, many apologies.  I've just realised that I put together an extensive set of changes to make the code work in the CentOS environment, but then failed to commit it :(.

Really sorry - I'll look at the changes I've made and see whether they address your concerns